docs/client/data.js

```diff
@@ -2976,7 +3157,9 @@ DocsData = {
     "summary": "Invoke a method passing an array of arguments."
   },
   "Meteor.call": {
+    "filepath": "ddp/livedata_connection.js",
```

Comment:
Yes, I think I'm prepared to make the compromise of only linking to one of the locations for now.